src/vpn-api/worker.ts

```diff
@@ -71,91 +46,191 @@ const getInstanceSubnet = (instanceId: number) => {
 };
 
 const worker = (instanceId: number) => {
-	logger.info(`worker-${instanceId} process started with pid ${process.pid}`);
+	const logger = getLogger('vpn', instanceId);
+
+	logger.notice(`process started with pid=${process.pid}`);
+
+	const fatalErrorHandler = (err: Error) => {
+		logger.emerg(`${err.message}`);
+		process.exit(2);
```

Comment:
Worth using `process.exitCode` instead, here? And in fact, maybe everywhere else where it's called.